packages/cli/src/commands/source/push/add.spec.ts

```diff
@@ -170,6 +176,26 @@ describe('source:push:add', () => {
       expect(ctx.stdout).toContain('Status code: 202 👌');
     });
 
+  test
+    .do(() => {
+      const largeFileSize = 5 * 1024 * 1024;
```

Comment:
So 5 mebibytes ?
If you mean 5 megabytes, it's 5e6

Reply:
nice